ArcGISRuntimeSDKQt_CppSamples/LocalServer/LocalServerMapImageLayer/LocalServerMapImageLayer.json

```diff
@@ -2,6 +2,12 @@
     "source" :  "LocalServerMapImageLayer",
     "description" : "LocalServerMapImageLayer.html",
     "thumbnail" : "LocalServerMapImageLayer.png",
-    "name" : "Local Server Map Image Layer"
+    "name" : "Local Server Map Image Layer",
+    "dataItems" : [        
+        {
+            "itemId" : " dee5d8060a6048a4b063484199a9546b",
```

Comment:
there is a leading space in the item id text string. that might be why you are having issues